pvlib/temperature.py

```diff
@@ -457,6 +457,108 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
+
+    temp_air : numeric
+        Ambient dry bulb temperature [C].
+
+    wind_speed : numeric, default 1.0
+        Wind speed in m/s measured at the same height for which the wind loss
+        factor was determined.  The default value 1.0 m/s is the wind
+        speed at module height used to determine NOCT. [m/s]
+
+    ir_down : numeric, default 0.0
+        Downwelling infrared radiation from the sky, measured on a horizontal
+        surface. [W/m^2]
+
+    u0 : numeric, default 25.0
+        Combined heat loss factor coefficient. The default value is one
+        determined by Faiman for 7 silicon modules.
+        :math:`\left[\frac{\text{W}/{\text{m}^2}}{\text{C}}\right]`
+
+    u1 : numeric, default 6.84
+        Combined heat loss factor influenced by wind. The default value is one
+        determined by Faiman for 7 silicon modules.
+        :math:`\left[ \frac{\text{W}/\text{m}^2}{\text{C}\ \left( \text{m/s} \right)} \right]`
+
+    sky_view : numeric, default 1.0
+        Effective view factor limiting the radiative exchange between the
+        module and the sky. For a tilted array the expressions
+        (1 + 3*cos(tilt)) / 4 can be used as a first estimate for sky_view
+        as discussed in [4]_. The default value is for a horizontal module. [-]
+
+    emissivity : numeric, default 0.88
+        Infrared emissivity of the module surface facing the sky. The default
+        value represents the middle of a range of values found in the
+        literature. [-]
+
+    Returns
+    -------
+    numeric, values in degrees Celsius
+
+    Notes
+    -----
+    All arguments may be scalars or vectors. If multiple arguments
+    are vectors they must be the same length.
+
+    If the input `ir_down` is not supplied the model output is the same as the
+    original Faiman model.
+
+    References
+    ----------
+    .. [1] Faiman, D. (2008). "Assessing the outdoor operating temperature of
+       photovoltaic modules." Progress in Photovoltaics 16(4): 307-315.
+
+    .. [2] "IEC 61853-2 Photovoltaic (PV) module performance testing and energy
+       rating - Part 2: Spectral responsivity, incidence angle and module
+       operating temperature measurements". IEC, Geneva, 2018.
+
+    .. [3] "IEC 61853-3 Photovoltaic (PV) module performance testing and energy
+       rating - Part 3: Energy rating of PV modules". IEC, Geneva, 2018.
+
+    .. [4] Driesse, A. et al (2022) "Improving Common PV Module Temperature
+       Models by Incorporating Radiative Losses to the Sky". SAND2022-11604.
+
+    '''
+    # Contributed by Anton Driesse (@adriesse), PV Performance Labs. Nov., 2022
+
+    # The following lines may seem odd since the values are probably scalar,
+    # but they serve an indirect and easy way of allowing lists and
+    # tuples for the other function arguments, and providing Series output.
+    u0 = np.asanyarray(u0)
+    u1 = np.asanyarray(u1)
+    emissivity = np.asanyarray(emissivity)
+
+    t_zero = np.array(-273.15)
```

Comment:
Should this constant be 298.15 to correspond with 25C (temperature at STC)?

Reply:
No, it's for converting to Kelvin.

Reply:
Lost me on that intent, on line 553. I think it would be more clear to just convert `temp_air` to `tempK` or whatever name you prefer.

Reply:
I agree that was not so clear.  Perhaps the rename will help.